### PR TITLE
Pil 663 - Copy updates to existing screens (Part 1)

### PR DESCRIPTION
**Pull request by olakunlekuye2**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -297,18 +294,18 @@ contactUPEByTelephone.change.hidden =  your response to whether we make contact
...
+NominateFilingMemberYesNo.title = Do you want to register a nominated filing member to report for this group?
+NominateFilingMemberYesNo.heading = Do you want to register a nominated filing member to report for this group?
 NominateFilingMemberYesNo.heading.caption = Group details
-NominateFilingMemberYesNo.p1 = This step is optional. If you do not enter the details for a nominated filing member, the ultimate parent entity (UPE) will be used as the default.
-NominateFilingMemberYesNo.insetText = You must have permission from the UPE to nominate a company as a filing member and be able to access the information required to file   Pillar 2 top-up tax on the group''s behalf.
+NominateFilingMemberYesNo.p1 = If you do not register a nominated filing member, we will use the ultimate parent entity (UPE) as the filing member.
+NominateFilingMemberYesNo.insetText = A nominated filing member must be nominated by the UPE and must be able to access the information required to report Pillar 2 top-up taxes on the group''s behalf.
 NominateFilingMemberYesNo.error.required = Select yes if there is a nominated filing member for this group
```
Comment (jassalrichy) [inline conf/messages.en:309]: Do we not need to update 'NominateFilingMemberYesNo.error.required' content to reflect the content amends to 'NominateFilingMemberYesNo.heading' ?

Comment (olakunlekuye2) [reply conf/messages.en:309]: Not according to the ticket

Comment (olakunlekuye2) [reply conf/messages.en]: Updated according to the updated ticket

**Review by jassalrichy — APPROVED** [review]

File: app/views/JourneyRecoveryStartAgainView.scala.html [hunk 2]
```diff
@@ -26,12 +26,8 @@
...
 
-    <p class="govuk-body">@messages("journeyRecovery.startAgain.guidance")</p>
-
     <p class="govuk-body">
-        @govukButton(
-            ButtonViewModel(messages("site.startAgain"))
-                .asLink(routes.IndexController.onPageLoad.url)
-        )
-    </p>
+        You must <a class="govuk-link" href="@routes.IndexController.onPageLoad.url">
```
Comment (jassalrichy) [inline app/views/JourneyRecoveryStartAgainView.scala.html]: 'You must' should be a message key

File: app/views/JourneyRecoveryContinueView.scala.html [hunk 3]
```diff
@@ -27,12 +27,7 @@
...
 
-    <p class="govuk-body">@messages("journeyRecovery.continue.guidance")</p>
-
     <p class="govuk-body">
-        @govukButton(
-            ButtonViewModel(messages("site.continue"))
-                .asLink(continueUrl)
-        )
-    </p>
+        You must <a class="govuk-link" href="@continueUrl">
```
Comment (jassalrichy) [inline app/views/JourneyRecoveryContinueView.scala.html]: 'You must' should be a message key

File: app/views/JourneyRecoveryStartAgainView.scala.html (hunk 2 above)
Comment (olakunlekuye2) [reply app/views/JourneyRecoveryStartAgainView.scala.html]: This view is no longer used and has been deleted

File: app/views/JourneyRecoveryContinueView.scala.html (hunk 3 above)
Comment (olakunlekuye2) [reply app/views/JourneyRecoveryContinueView.scala.html]: This view is no longer used and has been deleted

**Review by aashik-varahan — DISMISSED** [review]

**Review by javed-iqbal1 — APPROVED** [review]

**Pull request merged**